packages/workbox-streams/README.md

```diff
@@ -1 +1 @@
-This module's documentation can be found at https://developers.google.com/web/tools/workbox/reference-docs/latest/workbox.streams
+This module's documentation can be found at https://developers.google.com/web/tools/workbox/reference-docs/latest/workbox-streams
```

Comment:
I think this needs to be `module-workbox-streams`.

Reply:
I will take your word for that!

Reply:
It should be the same as how it currently works for `workbox-build`: https://developers.google.com/web/tools/workbox/reference-docs/latest/module-workbox-build